tests/datavolume_test.go

```diff
@@ -1119,7 +1119,7 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 				size:             "1Gi",
 				url:              vcenterURL,
 				dvFunc:           createVddkDataVolume,
-				eventReason:      common.AwaitingVDDK,
+				eventReason:      "Pending",
```

Comment:
Note for the reviewer: I changed this `eventReason` to pending since with populators, DataVolumes won't get boundCondition events.